backend/Search.hs

```diff
@@ -4,40 +4,45 @@
 
 module Main (main) where
 
-import System.IO
-import System.IO.Error
 import Control.Applicative
-import Control.Monad
 import Control.Concurrent (threadDelay)
-import Servant
-import Servant.Types.SourceT
-import Servant.HTML.Blaze (HTML)
-import Network.Wai.Handler.Warp
-import Data.Monoid
-import qualified Network.Socket
-import qualified Data.Streaming.Network
-import qualified Options.Applicative as Opt
-import System.Process
 import Control.DeepSeq (rnf)
 import Control.Exception hiding (Handler)
+import Control.Monad
```

Comment:
Well, strictly speaking, our coding style requires implicit imports, but, I guess, this is on @int-index since he doesn’t seem to be following it in this project anyway 🙄.

Reply:
Yeah, there's a lot that could be improved style-wise. But I've now added explicit imports anyway :nate: